WiFi101WebThingAdapter.h

```diff
@@ -40,15 +40,13 @@ enum ReadState {
 class WebThingAdapter {
 public:
   WebThingAdapter(String _name): name(_name), server(80), mdns(udp) {
-    // String url = "url=http://" + name + ".local";
-    serviceTxt = "\x13webthing=true"; // _" + url;
-    // serviceTxt[14] = (char)url.length();
+    serviceTxt = "\x13path=/";
```

Comment:
I wasn't sure what the `\x13` was for here.

Reply:
\x13 is the length of "webthing=true" although it should actually be \x0d (which explains a lot in hindsight).

To add the path the correct incantation is located around here:
https://github.com/arduino-libraries/ArduinoMDNS/blob/master/examples/WiFi/WiFiRegisteringServicesWithTxtRecord/WiFiRegisteringServicesWithTxtRecord.ino#L99

It was removed since the incorrect length \x13 was corrupting the packets